plugins/CoreAdminHome/lang/en.json

```diff
@@ -139,6 +139,17 @@
         "TrackingFailuresEmail1": "This is just to let you know that %s different kinds of tracking failures have occurred in the last days.",
         "TrackingFailuresEmail2": "To view all the failed tracking requests %1$sclick here%2$s.",
         "ImportFromGoogleAnalytics": "Import from Google Analytics",
-        "ImportFromGoogleAnalyticsDescription": "If you have a Google Analytics property and are switching to Matomo, you can import your existing data using the %1$sGoogleAnalyticsImporter%2$s plugin."
+        "ImportFromGoogleAnalyticsDescription": "If you have a Google Analytics property and are switching to Matomo, you can import your existing data using the %1$sGoogleAnalyticsImporter%2$s plugin.",
+        "SecurityNotificationEmailSubject": "Security Notification",
+        "SecurityNotificationRecoveryCodesShowedBody": "Someone viewed the two-factor authentication recovery codes of your account. If it was you, carry on. If you don't recognize this activity, please reset your password and check your two-factor authentication app or device.",
```

Comment:
You mean it should have more translations and we link them in the `getBody()` method?

Reply:
Something like:

```
'EmailBodyMessage' => 'Main message'
'IfItWasYouCarryOn' => 'If it was you, carry on. If you don't recognize this activity, please reset your password and check your two-factor authentication app or device.'
```
in the json files, and:
```
Piwik::translate('EmailBodyMessage') . Piwik::translate('IfItWasYouCarryOn')
```
in the php (or the equivalent in the twig files).

cc @sgiehl in case this is not preferred way.